{"frames":[{"func":"save(mx_env, mod_null_final, mod_fine_final, mod_meso_final, \n    mod_large_final, file = \"final_models/final_models&data.RData\")","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"gzfile(file, \"wb\")","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0}],"message":"Error in gzfile(file, \"wb\") : cannot open the connection\n"}